### ISSUE TITLE
Images doesn't have explicit height and width (Accessibility and CLS)

### ISSUE THREAD

**Issue by Ritvik-UX** [Subhradeep10/Fitnezz#16]
The images are not proportional and aspect ratio is not maintained while checking for responsiveness.
When you check it with [PageSpeed Insight](https://developers.google.com/speed/pagespeed/insights/) later, you will find this issue affecting Cumulative Layout Shift (CLS) that is a Core Web Vital.
So, it's better to fix it now :D

@Yusuf-gitCollab - It would be better if you fix this issue as you coded the responsiveness part of images.

Preferred Ratio = 16:9
Common Perfect Aspect Ratios below:

Choose the best sizes and apply in the images for different breakpoints.
And Apply this [code from stackoverflow with all img tags](https://stackoverflow.com/questions/65076805/image-elements-do-not-have-explicit-width-and-height/65077326).
![perfect aspect ratios](https://user-images.githubusercontent.com/73193822/133289397-b54d53b5-1942-4919-b662-180826964c1a.png)

**Comment by Subhradeep10** [comment]
@Yusuf-gitCollab Go ahead with the issue , if you have no problem

**Comment by Yusuf-gitCollab** [comment]
Okay I would just a sec

**Comment by Yusuf-gitCollab** [comment]
@Ritvik-UX actually if I set the width greater that 350px it would mess up with the responsive part of the site. Like in mobile screens images will overflow. But still I would try.

**Comment by Ritvik-UX** [comment]
@Yusuf-gitCollab - 
You can set image size to 256px x 144px as well. 
You can use [Aspect Ratio Calculator](https://calculateaspectratio.com/) for shortcut, or,
you  can Apply Ratio and Proportion (Maths) manually.
:D

**Comment by Yusuf-gitCollab** [comment]
@Ritvik-UX  oh well thank you. But I had one more confusion. The guide I was reading also told to use width and height as close as the original width and height. Another question I had is can simply adding the width and height attributes to the img tag in html solve the problem?

**Comment by Ritvik-UX** [comment]
@Yusuf-gitCollab - 
- Not exactly, Height as close as possible, but, the aspect ratio should be added properly.
- Yes you can add height and width in img src attribute.

You might be noticing that height gets small when you follow the proper Aspect Ratios pixels, thats fine.
You can add as much height and width but only factor is that their aspect ratio should be present in the chart above. (16:9, 1:3, 2:3) and its better if aspect ratios are kept consistent throughout.

This is crucial to six bcz, Google launches a core update once or twice a year, and in 2021, they launched Google Page Experience Update as the core update and fixing CLS one of the factors and is vital for SEO, UX and website ranking in SERPs.
Once site is active, it would important in future, so it's better to fix beforehand.
:D
